alphagov/paas-cf
# [#111946739] Parameterise zone ID, zone name, and ELB cert ARN

## What

This PR moves the `zone_id`, `zone_name` and `elb_whateveritis` parameters to being specified separately for both the "development" and "production" accounts. They're hardcoded in terraform `.tfvar` files for convenience.

Due to terraform quirks we need to declare the variables up front for cases where we're not actually using the `.tfvars` as it will throw errors - we use `dummy_default` as a placeholder, as it will be overridden by the `production.tfvars` or `development.tfvars` later.

The credentials for production are currently dummy values until [111676128](https://www.pivotaltracker.com/n/projects/1275640/stories/111676128) is complete.

## How to test

Spin up a concourse and an environment and it should deploy correctly, defaulting to the "development" credentials. Switching to production is achieved by defining the `ACCOUNT` environment variable as `production`, however testing this will currently explode in a hilarious fashion - the credentials are dummy values until the story to generate them has been finished.

## Who can review this

Anybody but @mtekel and @jonty
